.github/workflows/build.yml

```diff
@@ -1,51 +1,69 @@
 name: Build
+
 on:
   push:
     branches:
       - '*'
   pull_request:
     branches:
       - master
+
 jobs:
   build:
     strategy:
       matrix:
-        platform: [ubuntu-latest]
+        platform: [ubuntu-latest, windows-latest]
         otp-version: [21.3, 22.2]
+
     runs-on: ${{ matrix.platform }}
+
     container:
       image: erlang:${{ matrix.otp-version }}
+
     steps:
+    - name: Install Erlang
+      runs-on: windows-latest
+      run: choco install erlang
+
     - name: Checkout
       uses: actions/checkout@v2
+
     - name: Cache Hex packages
       uses: actions/cache@v1
       with:
         path: ~/.cache/rebar3/hex/hexpm/packages
         key: ${{ runner.os }}-hex-${{ hashFiles(format('{0}{1}', github.workspace, '/rebar.lock')) }}
         restore-keys: |
           ${{ runner.os }}-hex-
+
     - name: Cache Dialyzer PLTs
       uses: actions/cache@v1
       with:
         path: ~/.cache/rebar3/rebar3_*.plt
         key: ${{ runner.os }}-dialyzer-${{ hashFiles(format('{0}{1}', github.workspace, '/rebar.config')) }}
         restore-keys: |
           ${{ runner.os }}-dialyzer-
+
     - name: Compile
-      run: rebar3 compile
+      run: escript rebar3 compile
```

Comment:
Is there any way we can avoid prepending `escript`? Maybe automate the steps from https://www.rebar3.org/docs/getting-started and add them to the provisioning?